content/en/docs/tasks/traffic-management/mirroring/index.md

```diff
@@ -236,7 +236,7 @@ log entries for `v1` and none for `v2`:
     {{< /text >}}
 
     This route rule sends 100% of the traffic to `v1`. The last stanza specifies
-    that you want to mirror to the `httpbin:v2` service. When traffic gets mirrored,
+    that you want to mirror 100% of traffic from `httpbin:v1` to the `httpbin:v2` service. When traffic gets mirrored,
```

Comment:
This is better, but still not exactly correct. It's not mirroring traffic "from" `httpbin:v1`. It's mirroring the traffic that is being sent "to" `httpbin:v1`. More precisely, it's mirroring "all the incoming traffic", regardless of where's it's actually being sent. You could, for example, set up a rule to send 50% of traffic to v1, 50% to v2, and mirror 100% to v3. The mirror means what percentage of incoming traffic should go to a mirror destination. How about clarifying like this:
```suggestion
    that you want to mirror (i.e., also send) 100% of the same traffic to the
    `httpbin:v2` service. When traffic gets mirrored,
```